[MTM-50954] Change-screenshots-for-Enterprise-tenant-documentation-for-Managing-tenants-part (#834)
SoftwareAG/c8y-docs: content/users-guide/enterprise-tenant-bundle/managing-tenants.md

* Unhealthy - the number of inactive microservice instances
* Desired - the number of desired microservice instances
* Name - microservice instance name
* Restarts - the number of microservice instance restarts. 

{{< c8y-admon-info >}}

> Information about microservice instance name and number of restarts is displayed in case of at least one restart occurance.

Reviewer: ```suggestion
Information on the microservice instance name and the number of restarts is displayed in case of at least one restart.
```

* Unhealthy - the number of inactive microservice instances
* Desired - the number of desired microservice instances
* Name - microservice instance name

> * Restarts - the number of microservice instance restarts. _Note: information shown only if at least one restart occured._

Author: I have moved it to admon info, please check linguistic again :)
